src/html/shared/header.html

```diff
@@ -0,0 +1,23 @@
+<header class="c-header">
+    <button class="c-header__icon  c-icon  c-icon--vsmall">{{sprite("menu")}}</button>
+
+    <div class="c-header__logo">
+        <span class="u-hidden-visually">Shopiflight logo</span>
```

Comment:
instead of making the logo a background-image, try to make it have a physical element unless it's necessary to do so.

Reply:
helps with google search engine crawling images from your website.